test/Atmos/Parameterizations/Microphysics/KinematicModel.jl

```diff
@@ -50,8 +50,34 @@ using ClimateMachine.VTK
 
 using CLIMAParameters
 using CLIMAParameters.Planet: R_d, cp_d, cv_d, cv_v, T_0, e_int_v0, grav
-struct EarthParameterSet <: AbstractEarthParameterSet end
-const param_set = EarthParameterSet()
+
```

Comment:
What is the default output here? / Where are the diagnostics configured? 
I'm trying to compare this to e.g. [bomex.jl](https://github.com/CliMA/ClimateMachine.jl/blob/master/experiments/AtmosLES/bomex.jl) because you have this set up as an LES case. So if I wanted to run the KinematicModel.jl, where is the equivalent of main() from bomex.jl located? How do I specify the diagnostic output interval, format, etc. so that after I run it I can look at the output?
Thanks!

Reply:
It's not exactly the same as the LES configurations. Because of the kinematic setup I can't really use the LES config files that we have. I have to overwrite and redefine many things. I plan to have it in the repo as a test only. And then a tutorial will be a proper LES case. 
If you want I can walk you through how I coded the kinematic eddy model. But it's not how the other LES cases should be done. And it's not how the LES interface to microphysics will look like.